### ISSUE TITLE
document who to contact for infrastructure requests

### ISSUE THREAD

**Issue by benoitc** [erlef/infra-wg#7]
add informations about the way to contact the infrastructure group pon the WG website page and be more precise about what people can ask to us.

**Comment by benoitc** [comment]
related to #3

**Comment by starbelly** [comment]
Do you think we should only have one way contact method? Email, as an example, but only email.

**Comment by benoitc** [comment]
hrm it dpends for now, we can only receive private requests via mail but imo we shoudl also encourage open discussions so anyone from the community can participate. That's to say we should probably have both mails and tickets until we are on Github. 

Also it is easierto start a discussion on the ticket system anyway so we can track and trace requests in a simple manner. Thoughts?

**Comment by starbelly** [comment]
Yeah, my thoughts as far having one method is to avoid overlap and fragmentation. I'm fine with at  two and for the reasons you stated... In general open up tickets, if it's sensitive, email us.
